lib/src/Checkbox/checkbox.scss

```diff
@@ -38,5 +37,8 @@ $checkbox-padding: $checkbox-padding $checkbox-padding $checkbox-padding $checkb
                 color: $gray-300;
             }
         }
+        > .custom-control-label {
```

Comment:
this will make the cursor pointer even if the checkbox is disabled ?
Vanilla and bootstrap libraries doesn't have this are you sure you want to add it to react lib ?